coz-shop/src/components/Bookmark.js

```diff
@@ -1,9 +1,38 @@
+import Category from "./Category"
 import Menu from "./menu"
+import Product from "./Product"
+import { useState,useEffect } from "react"
+import Toast from "./Toast"
+const Bookmark = ({menuOn,menuHandle,bookmark,bookmarkHandle,toastEvent,filterResult}) =>{
+    const [focus , setFocus] = useState('All')
+   const [curruntArr , setCurruntArr] = useState(bookmark)
+   const addMessege = '북마크에 추가되었습니다.'
+   const delMessege = '북마크가 제거되었습니다.'
+   const focusHandle = (e) =>{
+      setFocus(e)
+   }
 
-const Bookmark = ({menuOn,menuHandle}) =>{
-    return <div>
+   useEffect(()=>{
+      if(focus==='All'){
+         setCurruntArr(bookmark)
+      }else{
+         setCurruntArr(
+            bookmark.filter((el)=>{
+            return el.type === focus
+         }))
+      }
+   },[focus,bookmark])
+   
+    return <div className="page_Container">
+        {toastEvent ? <Toast messege={filterResult? delMessege : addMessege} markOn={filterResult? false : true}/> : ''}
+        <Category focus={focus} focusHandle={focusHandle} />
+    <div>
         {menuOn? <Menu menuHandle={menuHandle}/> : ''}
     </div>
+    <div className="Product_Page_Container">
+   {curruntArr.length? curruntArr.map((el,index)=> <section className="Product_Container" key ={index}><Product obj={el} key ={el.id} bookmarkHandle={bookmarkHandle} bookmark={bookmark} /> </section>) : <section className="bookmark_help">새로운 상품을 북마크 해보세요</section>}
```

Comment:
```suggestion
   {curruntArr.length? curruntArr.map((el,index)=> <section className="Product_Container" key ={index}><Product obj={el} key ={el.id} bookmarkHandle={bookmarkHandle} bookmark={bookmark} /> </section>) : <section className="bookmark_help">새로운 상품을 북마크 해보세요</section>}
```

클래스이름이 대문자 소문자가 혼용되는 부분이 있습니다 ㅠ
가능하면 통일해서 사용해주시면 좋겠습니다.

Reply:
이거는 지금 다 고치기 너무 힘들 거 같고 다음부터 신경 써서 써보도록 하겠습니다 ㅠㅠ